doc/tutorial/ApiType.lhs

```diff
@@ -389,3 +389,30 @@ One example for this is if you want to serve a directory of static files along
 with the rest of your API. But you can plug in everything that is an
 `Application`, e.g. a whole web application written in any of the web
 frameworks that support `wai`.
+
+Be mindful! This library works by pattern-matching the different routes that are
+composed using `:<|>`. `Raw`, as an escape hatch, matches any route that hasn't
+been matched by previous patterns. Therefore, any subsequent route will be silently
+ignored.
+
+``` haskell
+type UserAPI14 = Raw
+            :<|> "users" :> Get '[JSON] [User]
+                 -- In this situation, the /users endpoint
+                 -- will not be reachable because the Raw
+                 -- endpoint matches requests before
+```
+A simple way to avoid this pitfall is to either use `Raw` as the last
+definition, or to always have it under a directory.
```

Comment:
Here and in the Haskell comment below, I'd probably use a slightly more generic vocabulary than "directory", as there's usually no guarantee that this indeed corresponds to any sort of directory. "under a static path", "under a specific endpoint", or something along those lines...?

Reply:
I actually didn't know what to call it, `static path` is a great suggestion.
I will also be more explicit in the example that the static path is the `/file/` bit, since there doesn't seem to be a widely accepted term for that.